### PR TITLE
Added "About this log" to the tasklist

### PR DESCRIPTION
**Pull request by roobottom**

### CODE REVIEW

File: app/views/logs/log.njk [hunk 1]
```diff
@@ -20,7 +20,7 @@
       <h2 class="govuk-heading-s govuk-!-margin-bottom-2">This log is incomplete</h2>
       <p class="govuk-body govuk-!-margin-bottom-7">
         You have completed 0 of {{ data.sections | length }} sections.<br>
-        Skip to next incomplete section: <a href="{{ nextSection(log.id).href }}">{{ nextSection(log.id).text }}</a>
+        Skip to next incomplete section: <a href="#item-{{ nextSection(log.id).text | lower | replace(" ", "-") }}">{{ nextSection(log.id).text }}</a>
```
Comment (paulrobertlloyd) [inline app/views/logs/log.njk:23]: Can you not do this?

```suggestion
        Skip to next incomplete section: <a href="#item-{{ nextSection(log.id).id }}">{{ nextSection(log.id).text }}</a>
```

File: app/components/task-list/_task-list.scss [hunk 2]
```diff
@@ -49,6 +49,19 @@
...
+}
+
+.app-task-list__additional {
+  display: block;
+  clear: left;
+  @include govuk-clearfix;
+  @include govuk-responsive-margin(7, "top");
+  >* {
+    margin-bottom: 0;
+  }
```
Comment (paulrobertlloyd) [inline app/components/task-list/_task-list.scss:64]: What are these additions for?

Comment (paulrobertlloyd) [inline app/components/task-list/_task-list.scss:64]: What is this class for?

**Review by paulrobertlloyd — APPROVED** [review]
Love it!

**Comment by paulrobertlloyd** [comment]
I’m about the accessibility of moving focus to a list item, and not the item’s link. No idea which is the better approach.

File: app/components/task-list/_task-list.scss (hunk 2 above)
Comment (roobottom) [reply app/components/task-list/_task-list.scss:64]: I'll remove it.

File: app/views/logs/log.njk (hunk 1 above)
Comment (roobottom) [reply app/views/logs/log.njk:23]: What do you mean by "not do this"?

**Comment by roobottom** [comment]
> I’m curious about the accessibility of moving focus to a list item, and not the item’s link. No idea which is the better approach.

We tested this in HMRC, it gave users of screen readers the context of the link and label together.

File: app/views/logs/log.njk (hunk 1 above)
Comment (paulrobertlloyd) [reply app/views/logs/log.njk:23]: Doesn’t each section have a kebab-cased `id` value already, meaning you don’t need to convert the name to kebab case?

Comment (roobottom) [reply app/views/logs/log.njk]: not in the `nextSection` obj - but good call about separating concerns. I made a change.

**Pull request merged**